ospfd/ospf_vty.c

```diff
@@ -1567,6 +1567,8 @@ static int ospf_area_nssa_cmd_handler(struct vty *vty, int argc,
 			ospf_area_no_summary_unset(ospf, area_id);
 	}
 
+	/* Flush the external LSA */
```

Comment:
Pl update the comment  " Flush the external LSA for given area "

Reply:
updated